scripts/core/kernel.yml

```diff
@@ -158,12 +160,15 @@ params:
     - type: $x_kernel_group_info_t
       name: propName
       desc: "[in] name of the work Group property to query"
-    - type: "size_t"
+    - type: "uint32_t"
```

Comment:
Should this be `uint32_t` or `size_t`, it's a memory size in bytes right?

Reply:
👍